### ISSUE TITLE
Re: Install - composer install got errors, composer update did not.

### ISSUE THREAD

**Issue by swizca** [chimo/gs-emojiShortnames#1]
Following install instructions, composer install produced errors (emojione).

Running composer update did not.

Perhaps change 'composer install' in instructions to 'composer update' - it feels like update will install if not present, making things happier all around.

**Comment by chimo** [comment]
That's odd. Do you remember what the errors were?

**Comment by swizca** [comment]
Not odd for the seriously defective environment I was in. :-)

Based on our discussion in gsReverse, I finally got around to re-scripting my gs plugin installs to deploy into local/plugins instead of plugins, including your kindly noted 'git <repository> <desired directory>'.

Note: One gotcha that bit me was repository name of noticePreview failed for lack of NoticePreview.

So:
(1) git install to local/plugins; working config.php
(2) mv plugins/dir plugins/dir.old
(3) refresh social page seeking no change, schema check.

- schema checks fail for different reasons. One - mysterious, two - noticePreview unhappiness, three - mysterious can't find vendor/something.php file.

In pursuing issues, got to github emojiShortnames install note to realize I hadn't done a corresponding composer install in the new directory. To get errors, where it noted a composer update was needed.

Ran composer update, it got happy, composer install noted nothing to do, and this aspect of the transition concluded happily. (Next step of npm install was already happy, due to prior install in prior directory. [Being a different form of beastie than composer.])

If 'composer update' does not trigger a 'composer install' if needed, then ignore this OP and close it. If it does, noting it in install doc may reduce unexpected results for some - like myself, here.

**Comment by chimo** [comment]
I tried the installation and did get a warning about the lockfile being out of date, so I pushed a commit that should fix the problem.

Both `composer install` and `composer update` end up installing the dependencies, but in a slightly different way. This explains it better than I could: https://stackoverflow.com/a/33052263

With the change I pushed today `npm install` should work fine. Let me know if you find otherwise.

> Note: One gotcha that bit me was repository name of noticePreview failed for lack of NoticePreview.

I added a [README](https://github.com/chimo/gs-noticePreview#notice-preview) to the "noticePreview" repository with installation commands similar to the ones here. Thanks for the reminder!
